docs/destination.md

```diff
@@ -92,3 +92,102 @@ The Timeplus source plugin for Airbyte is in the early stage. Please contact us
 ## Trigger Actions via webhook{#webhook}
 
 You can also add automations to trigger other systems to take actions when Timeplus finds any real-time insights. Simply choose the **Webhook** as the action type and optionally set a message body (by default, the entire row will be encoded as a JSON document and sent to the webhook). You can use this approach to perform rule-based automation without human interaction, such as swapping a overheated equipment, scaling up to scaling down the server farm, or reminder users on slack, etc. Please check [this blog](https://www.timeplus.com/post/build-a-real-time-security-app-in-3-easy-steps) for real-world examples.
+
+
+## Sink API
+In case calling API to create a sink, here are the reference
+
+### kafka
+
+refet to [https://kafka.apache.org/](https://kafka.apache.org/)
+
+| Property                | Required    | Description                                               | Default |
+|-------------------------|---------|-----------------------------------------------------------|---------|
+| brokers                   | yes | Specifies the list of broker addresses. This is a comma-separated string. such as `kafka1:9092,kafka2:9092,kafka3:9092`| | |
+| topic                     | yes | Specifies the Kafka topic to send data to.              | |
+| batch_count               | no  | Specifies the Kafka offset configuration.    support `latest,earliest`            |`1000`|
+| data_type                 | yes | Specifies the data type to use for creating the stream.   support `json`,` `,`protobuf` | | 
```

Comment:
the empty string ` ` is not obvious in the rendered doc

Reply:
Maybe we don't need to mention protobuf for now? Right now the protobuf is too PoC. For example, in order to use protobuf, the user need to pass `protobuf_schema` and `protobuf_message` that matches the output of Sink. However, it is very difficult for the user to write the schema by themselves. We should dynamically analyze the SQL result and create the schema for them